h2o-r/tests/testdir_algos/grid/runit_grid_recovery_dir.R

```diff
@@ -22,6 +22,13 @@ test.grid.resume <- function() {
   # check the recovery dir is empty after grid success
   print(list.files(export_dir))
   expect_true(length(list.files(export_dir)) == 0)
+  grid_path <- h2o.saveGrid(export_dir, grid_id, save_params_references=TRUE)
+  h2o.removeAll()
+
+  grid <- h2o.loadGrid(grid_path, load_params_references = TRUE)
+  h2o.resumeGrid(grid@grid_id)
```

Comment:
Test resume with new folder as well ?

Reply:
good idea, it is difficult to test with the current code and it would a lot of wasted work, since my next PR will make this much easier. in my next PR I will add such tests

Reply:
Agreed.